packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -86,13 +92,30 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
   /// Scrollbar will be rectangular if [radius] is null.
   final Radius radius;
 
-  /// The smallest size the scrollbar can shrink to when the total scrollable
-  /// extent is large and the current visible viewport is small, and the
-  /// viewport is not overscrolled. Mustn't be null.
+  /// The amount of space by which to inset the scrollbar's start and end, as
+  /// well as its side to the nearest edge, in pixels.
+  ///
+  /// This is typically set to the current [MediaQueryData.padding] to avoid
+  /// partial obstructions such as display notches. If you want additonal
+  /// margins around the scrollbar, see [mainAxisMargin] or [crossAxisMargin].
+  ///
+  /// Defaults to [EdgeInsets.zero]. Must not be null and offsets from all four
+  /// directions must be greater than or equal to zero.
+  final EdgeInsets padding;
+
+  /// The preferred smallest size the scrollbar can shrink to when the total
+  /// scrollable extent is large and the current visible viewport is small, and the
+  /// viewport is not overscrolled.
+  ///
+  /// Mustn't be null and the value is typically greater than or equal to
+  /// [minOverscrollLength].
   final double minLength;
 
-  /// The smallest size the scrollbar can shrink to when viewport is
-  /// overscrolled. Mustn't be null.
+  /// The preferred smallest size the scrollbar can shrink to when viewport is
```

Comment:
In case we say 'preferred' describe when do things fall out of the preferred behavior.